gtfs/spec/en/reference.md

```diff
@@ -407,14 +407,14 @@ File: **Optional**
 
 Primary Key (`fare_product_id`, `fare_media_id`)
 
-To describe the different types of tickets or fares that can be purchased by riders.
+To describe the different types of tickets or fares, including those directly purchasable by riders and integral virtual entities like transfer costs.
 
 |  Field Name | Type | Presence | Description |
 |  ------ | ------ | ------ | ------ |
-| `fare_product_id` | ID | **Required** | Identifies a fare product. |
+| `fare_product_id` | ID | **Required** | Identifies a fare product.<br><br>Multiple records in [fare_products.txt](#fare_productstxt) may have the same `fare_product_id`, in which case all fare products with that ID will be applied when referenced from another file. |
 | `fare_product_name` | Text | Optional | The name of the fare product as displayed to riders. |
 |  `fare_media_id` | Foreign ID referencing `fare_media.fare_media_id` | Optional |  Identifies a fare media that can be employed to use the fare product during the trip. When `fare_media_id` is empty, it is considered that the fare media is unknown.|
-| `amount` | Currency amount | **Required** | The cost of the fare product. May be negative to represent transfer discounts. May be zero to represent a fare product that is free.|
+| `amount` | Currency amount | **Required** | The cost of the fare product. May be negative to represent transfer discounts. May be zero to represent a fare product that is free.<br><br>Different records with the same `fare_product_id` may have different amount values, indicating variations in pricing.|
```

Comment:
Yes, it seems it would be better to explain within the description of the `fare_product_id` field rather than individually in all non-primary key fields. I will remove the description from the amount field.